docs/source/installation/login.md

```diff
@@ -94,46 +104,47 @@ qhub keycloak -c qhub-config.yaml adduser <username> <password>
 
 This will create a user `<username>` with the initial password provided. Omit the password completely if you are using GitHub or Auth0.
 
-> NOTE: This will also add the user to 'analyst' group.
+> NOTE: This will also add the user to the 'analyst' group.
 
 ## Login to QHub
 
-Your new user can now log into the main QHub dashboard (not Keycloak's admin console).
+Your new user can now log in to the main QHub dashboard (not Keycloak's admin console).
 
 Visit <https://myqhubsite.com/> (or whatever domain you have chosen for your QHub).
 
-Click 'Sign in with Keycloak'.
+Click 'Sign in with Keycloak'. This will take you to the login form:
 
 ![QHub - Log in to Keycloak page](../images/keycloak_qhub_login.png)
 
 If you chose GitHub or Auth0 login, click the 'GitHub' button to be taken to a GitHub login page and single-sign-on from there (as shown in the screenshot above). Otherwise, if you
-choose 'Password\` based authentication, enter the username and password you chose when you added a user to QHub above.
+chose 'Password' based authentication, enter the username and password you chose when you added a user to QHub above.
 
 ## Groups
 
-Groups represent a collection of users that perform similar actions and therefore require the similar permissions. By default, QHub is deployed with the following groups: 'admin',
-'developer', 'analyst' and 'viewer'.
+Groups represent a collection of users that perform similar actions and therefore require similar permissions. By default, QHub is deployed with the following groups: 'admin',
+'developer', 'analyst' and 'viewer' (in roughly descending order of power).
```

Comment:
Why isn't 'viewer' in the table below?

Reply:
I think we don't even have the viewer anymore, but I would need to do a fresh deploy to be sure.